app/src/main/java/com/example/keybindhelper/RecyclerViewAdapters/GroupAdapter.java

```diff
@@ -139,16 +139,16 @@ public void onBindViewHolder(@NonNull GroupViewHolder holder, int position) {
                 d.cancel();
                 ArrowProvider ap=new ArrowProvider(context);
                 ap.directionClicked=isUp -> {
-                    int indx= CurrentProjectManager.CurrentProject.Groups.indexOf(group);
-                    System.out.println(indx);
+                    int index= CurrentProjectManager.CurrentProject.Groups.indexOf(group);
+                    System.out.println(index);
```

Comment:
Changed indx to index since it is not great practice to abbreviate. Source: https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/general-naming-conventions

"DO NOT use abbreviations or contractions as part of identifier names."

Reply:
- Good idea.  This was covered in a previous code review.  When that occurs, first in wins.